src/EditorFeatures/Core.Wpf/InlineRename/CommandHandlers/RenameCommandHandler_EscapeHandler.cs

```diff
@@ -1,19 +1,21 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
-using Microsoft.CodeAnalysis.Editor.Commands;
+using Microsoft.VisualStudio.Commanding;
+using Microsoft.VisualStudio.Text.Editor.Commanding.Commands;
 using Microsoft.VisualStudio.Text.Editor;
+using VSCommanding = Microsoft.VisualStudio.Commanding;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.InlineRename
 {
-    internal partial class RenameCommandHandler : ICommandHandler<EscapeKeyCommandArgs>
+    internal partial class RenameCommandHandler : IChainedCommandHandler<EscapeKeyCommandArgs>
```

Comment:
>IChainedCommandHandler [](start = 50, length = 22)

some of these can be simple handlers, but it's too late and too much code churn at this point so I prefer to keep it chained for now and convert to simple next iteration.